File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int c=1;
        int prev=nums[0];
        for(int i=1;i<nums.size();i++){
            if(nums[i]!=prev){
                nums[c]=nums[i];
                prev=nums[i];
                c++;
            }
        }
        return c;
    }
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cs
public class Solution {
    public int RemoveDuplicates(int[] nums) {
        HashSet<int> hashSet=new();
        int k=0;
        for(int i=0;i<nums.Length;i++){
            if(!hashSet.Contains(nums[i])){
                hashSet.Add(nums[i]);
                nums[k]=nums[i];
                k++;
            }
        }
        return k;
    }
}